requirements-dev.txt

```diff
@@ -32,6 +32,7 @@ pytest-xdist
 pytest!=7.1.0
 pyyaml
 torch>=2.1
+transformers==4.37.2
```

Comment:
I found this recent change breaks docker image build, as onnx-script wants 4.37.1, while pytorch/benchmark requires 4.38.1. Do we know if this is a hard requirements to set to 4.37.2, or it's just >=4.37.2?

Reply:
For the time being, let's do as you say.